src/shellCommands/ocSdkCommands.ts

```diff
@@ -279,51 +291,86 @@ export class OcSdkCommand {
   }
 }
 
+
 async function getJsonData(galaxyUrl: string, galaxyNamespace: string): Promise<any> {
-  let jsonData: any;
-  const urlScheme = vscode.Uri.parse(galaxyUrl).scheme;
+  const apiUrl =  `${galaxyUrl}/api/v3/plugin/ansible/content/published/collections/index/${galaxyNamespace}/operator_collection_sdk/versions/`;
+  const legacyApiUrl = `${galaxyUrl}/api/internal/ui/repo-or-collection-detail/?namespace=${galaxyNamespace}&name=operator_collection_sdk`;
+  const galaxyResponse = getRequest(apiUrl);
+  const legacyGalaxyResponse = getRequest(legacyApiUrl);
+  return Promise.all([galaxyResponse, legacyGalaxyResponse]).then((responses) => {
+    if (responses[0] !== undefined) {
+      return responses[0];
+    }
+    if (responses[1] !== undefined) {
+      return responses[1];
+    }
+    return undefined; 
+  }).catch((e) => {
+    return e;
+  });
+}
+
+async function getRequest(apiUrl: string): Promise<string | undefined> {
+  const urlScheme = vscode.Uri.parse(apiUrl).scheme;
   if (urlScheme === "https") {
-    jsonData = await new Promise((resolve, reject) => {
+    return new Promise<string | undefined>((resolve, reject) => {
       https
```

Comment:
this duplicated code could be replaced in a single one if we add a variable:
`const httpType = urlScheme === "https" ? require("https") : require("http");`

 and then we can access it  like: 
`      httpType.get(apiUrl,(resp: any) => {...})  
`